### ISSUE TITLE
line 243 : out of memory. etherum mining

### ISSUE THREAD

**Issue by JustFragger** [ethereum-mining/ethminer#1952]
**Hello Mining Community !**
I wanna start mining again on ethminer ,but still im getting this error `CUDA error in func 'ethash_cuda_miner::init' at line 243 : out of memory.`
I tried increasing my Virtua memory to 24GB but it doesnt worked for me :/

Im using :
Win 10
I7 7th gen processor
RAM : 8GB
GPU : Nvidia Geforce GTX 950m 4GB

Full log here:

```
C:\Users\ME\Desktop\ethminer-0.9.41-genoil-1.1.7>set GPU_FORCE_64BIT_PTR=0

C:\Users\MEDesktop\ethminer-0.9.41-genoil-1.1.7>set GPU_MAX_HEAP_SIZE=100

C:\Users\ME\Desktop\ethminer-0.9.41-genoil-1.1.7>set GPU_USE_SYNC_OBJECTS=1

C:\Users\ME\Desktop\ethminer-0.9.41-genoil-1.1.7>set GPU_MAX_ALLOC_PERCENT=100

C:\Users\ME\Desktop\ethminer-0.9.41-genoil-1.1.7>set GPU_SINGLE_ALLOC_PERCENT=100

C:\Users\ME\Desktop\ethminer-0.9.41-genoil-1.1.7>ethminer -U -S eth.pool.minergate.com:45791 -O "MY_EMAIL"
Genoil's ethminer 0.9.41-genoil-1.1.7
=====================================================================
Forked from github.com/ethereum/cpp-ethereum
CUDA kernel ported from Tim Hughes' OpenCL kernel
With contributions from nicehash, nerdralph, RoBiK and sp_

Please consider a donation to:
ETH: 0xeb9310b185455f863f526dab3d245809f6854b4d

Using grid size 8192, block size 128

  i  17:31:57|main  Connecting to stratum server eth.pool.minergate.com:45791
  i  17:31:58|stratum  Connected to stratum server eth.pool.minergate.com : 45791
  i  17:31:58|stratum  Starting farm
  i  17:31:58|stratum  Subscribed to stratum server
  i  17:31:58|stratum  Authorized worker MY_EMAIL
  i  17:31:58|stratum  Received new job #dbb10f63
  i  17:31:58|cudaminer0  set work; seed: #1e4d8477, target: #000010c6f7a0
  i  17:31:58|cudaminer0  Initialising miner...
  m  17:31:59|main  Mining on PoWhash #dbb10f63 : 0.00MH/s [A0+0:R0+0:F0]
Using device: GeForce GTX 950M (Compute 5.0)
  m  17:32:01|main  Mining on PoWhash #dbb10f63 : 0.00MH/s [A0+0:R0+0:F0]
  i  17:32:06|stratum  Received new job #f430859e
CUDA error in func 'ethash_cuda_miner::init' at line 243 : out of memory.

```

**Comment by AndreaLanfranchi** [comment]
You're reporting from the original cpp-ethereum which is no longer active.
Please download any newer release from the releases section of this repo and retry.

**Comment by AndreaLanfranchi** [comment]
In addition ... if that's the only card you're trying to mine with, the out-of memory error may be correct.
Gtx 950m have 4Gb memory ... consider part of it is used by Windows thus making less available for the DAG which is now almost 3.4 Gb

**Comment by JustFragger** [comment]
Okay I"m closing this issue because i don"t have enought memory in GPU card
